test/built-ins/String/prototype/indexOf/S15.5.4.7_A4_T1.js

```diff
@@ -12,21 +12,17 @@ includes: [$FAIL.js]
 
 var __obj = {toString:function(){return "\u0041B";}}
 var __obj2 = {valueOf:function(){throw "intointeger";}}
-var __str = {str__:"ABB\u0041BABAB"};
+var __str = "ABB\u0041BABAB";
 
 //////////////////////////////////////////////////////////////////////////////
 //CHECK#1
-with(__str){
-    with(str__){
```

Comment:
It seems to be some case of: Let's use additional language features to increase the test surface. :confused: